### PR TITLE
Clarify Kserve api in Readme 

### LINKED ISSUE #1721: Clarify that TorchServe supports KServe V2 Inference API
### 🚀 The feature

Hey folks - I was wondering whether TorchServe has considered supporting the Inference V2 API proposed by KServe (https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/required_api.md) It seems Triton, TFServing, and MLServer all support this API. 

Please let me know if I'm missing that this is already on a public roadmap.

### Motivation, pitch

It seems KServe V2 Prediction API is on the cusp of being a standard across the major inference servers. This would make it really easy to generalize inference formats

### Alternatives

Stay only with support of KServe V1 protocol

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by SangamSwadiK**
Clarify Kserve protocol versions supported in Readme.

## Description

Clarify Kserve protocol versions supported by torchserve, in Readme
Fixes #1721

## Type of change
- [x] This change requires a documentation update ( Readme)

## Feature/Issue validation/testing
Not applicable.

## Checklist:

- [x] Did you have fun?
- [x] Have you made corresponding changes to the documentation?

Please let me know if it needs to be rephrased or documented differently,
Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Refer to [torchserve docker](docker/README.md) for details.
   * [Kubeflow](https://v0-5.kubeflow.org/docs/components/pytorchserving/)
   * [MLflow](https://github.com/mlflow/mlflow-torchserve)
   * [Sagemaker](https://aws.amazon.com/blogs/machine-learning/serving-pytorch-models-in-production-with-the-amazon-sagemaker-native-torchserve-integration/)
-  * [Kserve](https://kserve.github.io/website/modelserving/v1beta1/torchserve/)
+  * [Kserve](https://kserve.github.io/website/modelserving/v1beta1/torchserve/) (Supports both v1 and v2 api)
```
Comment (msaroufim) [inline README.md:46]: How about this instead

```suggestion
  * [Kserve](https://kserve.github.io/website/modelserving/v1beta1/torchserve/) : Supports both v1 and v2 API
```

**Review by msaroufim — CHANGES_REQUESTED** [review]
Thanks for this change - I'd suggest we mention our support for the v2 API in 2 more places

`README.md`
`docs/management_api.md`

File: README.md (hunk 1 above)
Comment (msaroufim) [reply README.md]: Ok one last thing here seeing as how we're only linking to v1beta1 we should also link to v2 here 

`Kserve: Supports both [v1](v_1_link) and [v2](v_2_link) API

**Review by msaroufim — APPROVED** [review]

File: docs/management_api.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ The Management API listens on port 8081 and is only accessible from localhost by
 
 Similar to the [Inference API](inference_api.md), the Management API provides a [API description](#api-description) to describe management APIs with the OpenAPI 3.0 specification.
 
+Alternatively, if you want to use KServe, TorchServe supports both v1 and v2 API. For more details please look into this [kserve/README.md](https://github.com/pytorch/serve/tree/master/kubernetes/kserve)
```
Comment (maaquib) [inline docs/management_api.md:16]: `kserve/README.md` as the hyperlink text can be confusing as someone might try to look up a kserve directory which don't have.

```suggestion
Alternatively, if you want to use KServe, TorchServe supports both v1 and v2 API. For more details please look into this [kserve documentation](https://github.com/pytorch/serve/tree/master/kubernetes/kserve)
```

**Review by msaroufim — APPROVED** [review]

**Review by agunapal — APPROVED** [review]
Looks good

**Pull request merged**